src/sonic-ctrmgrd/ctrmgr/ctrmgr_iptables.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
```

Comment:
can you check the hostcfgd, there are nat rules there, do we have conflict there?

Reply:
i suggest to centralize the control plane acl into one place. hostcfgd.

Reply:
Checked this in-depth with @prsunny. 

Currently, hostcfgd touches iptables for “mangle” only, whereas we need to touch nat table. So we can't use hostcfgd.

The cacld manages nat table based on ACL_TABLE & ACL_RULE. But we can't add our rules to ACL_TABLE/RULE, as it is tightly controlled by graph data and any discrepancy will remove the rules that is non-existent in graph.

Per discussion, it is suggested that we keep our IP table update outside cacld too. In other words, let ctrmgrd do that.

NOTE: Each cacld restart, clears iptable "nat" and re-create based on ACL_TABLE & RULE. So to retain our rules, ctrmgrd is made dependent on cacld and start after cacld, and restart for every cacld restart.